Configuration/Settings.yaml

```diff
@@ -94,7 +94,7 @@ Neos:
       #################################
       # INTERNAL CONFIG (no API)
       #################################
-      frontendDevelopmentMode: false
+      frontendDevelopmentMode: true
```

Comment:
Here you accidentally committed a config. BTW, you can create a config file in your project and change the setting there instead of changing it in the package itself ;)

Reply:
Thanks :). I removed the change from the PR.